src/settings/dto/create-setting.dto.ts

```diff
@@ -0,0 +1,6 @@
+export class CreateSettingDto {
+  readonly name: string;
+  readonly data_type: string;
+  readonly account_id: number;
+  readonly value: string;
+}
```

Comment:
this class is missing validations. need to verify if the given inputs are in the right format.